bin/pygrb/pycbc_pygrb_minifollowups

```diff
@@ -0,0 +1,205 @@
+    node.new_output_file_opt(workflow.analysis_time, '.png',
+                             '--output-file', tags=extra_tags)
+    # Quantity to be displayed on the y-axis of the plot
+    node.add_opt('--y-variable', snr_type)
+    if ifo is not None:
+        node.add_opt('--ifo', ifo)
+    reset_central_time = shift_time - central_time
+    # Horizontal axis range the = prevents errors with negative times
+    x_lims = str(-5.+reset_central_time)+','+str(reset_central_time+5.)
+    node.add_opt('--x-lims='+x_lims)
+    # Plot title
+    if ifo is not None:
+        title_str = "'%s SNR at %.3f (s)'" %(ifo, central_time)
+        node.add_opt('--central-time', central_time)
+    else:
+        title_str = "'%s SNR at %.3f (s)'" %(snr_type.capitalize(), central_time)
+        node.add_opt('--central-time', central_time)
+    node.add_opt('--plot-title', title_str)
+
+    # Add job node to workflow
+    workflow += node
+
+    return node.output_files
+
+
+# =============================================================================
+# Main script starts here
+# =============================================================================
+parser = argparse.ArgumentParser(description=__doc__[1:])
+parser.add_argument('--version', action='version', version=pycbc.version.git_verbose_msg)
+parser.add_argument("-v", "--verbose", default=False, action="store_true",
+                    help="Verbose output")
+parser.add_argument('--followups-file',
+                    help="HDF format file containing trigger/injections to follow up")
+#parser.add_argument('--instrument', help="Name of interferometer e.g. H1")
+parser.add_argument('--wiki-file',
+                    help="Name of file to save wiki-formatted table in")
+parser.add_argument('--output-map')
+parser.add_argument('--transformation-catalog')
+parser.add_argument('--output-file')
+wf.add_workflow_command_line_group(parser)
+wf.add_workflow_settings_cli(parser)
+args = parser.parse_args()
+
+init_logging(args.verbose, format="%(asctime)s: %(levelname)s: %(message)s")
+
+workflow = wf.Workflow(args, args.workflow_name)
+
+wf.makedir(args.output_dir)
+
+# Create a FileList that will contain all output files
+layouts = []
+
+# Read the file with the triggers/injections to follow up
+logging.info('Reading list of triggers/injections to followup')
+fp = h5py.File(args.followups_file, "r")
+
+# Initialize a wiki table and add the column headers
+if args.wiki_file:
+    wiki_file = os.path.join(args.output_dir, args.wiki_file)
+    add_wiki_row(wiki_file, fp.keys())
+
+# Trigger time
+seg_dir = workflow.cp.get('workflow', 'segment-dir')
+trig_time = get_grb_time(seg_dir)
+
+# Establish the number of follow-ups to perform
+num_events = int(workflow.cp.get_opt_tags('workflow-minifollowups',
+                                          'num-events', ''))
+num_events = min(num_events, len(fp['BestNR'][:]))
+
+# Determine ifos used in the analysis
+trig_file = workflow.cp.get('workflow', 'trig-file')
```

Comment:
This is probably a FIXME for the future, but it's not best practice to send files through like this. It assumes a shared file system (which is normally true in practice, but doesn't have to be). Better to have this as an explicit input file. The workflow generator itself always runs on a head node, so can assume this safely, but this code will run on a compute node.